Medals/BaseMedals.as

```diff
@@ -0,0 +1,55 @@
+class AuthorMedal : Medal
+{
+	bool IsVisible() override
+	{
+#if TURBO
+		return showTmaster;
+#else
+		return showAuthor;
+#endif
+	}
+
+	protected string GetName() override
+	{
+#if TURBO
+		return tmasterText;
+#else
+		return authorText;
+#endif
+	}
+
+	protected int GetScoreAsync(CGameCtnChallenge@ map) override { return map.TMObjective_AuthorTime; }
+
+	protected vec3 GetIconColor() override
+	{
+#if TURBO
+		return vec3(0, 1, 0x11/255.0f);
+#else
+		return vec3(0, 0x77/255.0f, 0x11/255.0f);
```

Comment:
I would probably write a helper that converts an int like `0x007711` here to a vec3. 🤔

Reply:
The benefit of doing it this way is that `0x77/255.0f` is essentially 2 constants and can be inlined fairly easily, reducing the amount of overhead. It's super minimal, and a helper would definitely be cleaner though.

Reply:
Ah, yeah, that makes sense